src/sentry/tasks/merge.py

```diff
@@ -188,8 +189,27 @@ def merge_groups(
                     times_seen=F("times_seen") + group.times_seen,
                     num_comments=F("num_comments") + group.num_comments,
                 )
-            except DataError:
-                pass
+            except DataError as e:
+                logger.warning(
+                    "tasks.merge.times_seen_update_failure",
+                    extra={
+                        "new_group_id": new_group.id,
+                        "old_group_id": group.id,
+                        "new_times_seen": new_group.times_seen,
+                        "old_times_seen": group.times_seen,
+                        "attempted_times_seen": new_group.times_seen + group.times_seen,
+                        "error_message": str(e),
```

Comment:
why don't we use sentry.captureException(e,level=warning) for this if we want the error? 

```
with sentry_sdk.push_scope() as scope:
    scope.set_extra("new_group_id", new_group.id)
    ...
    sentry_sdk.capture_exception(Exception("Checkout failed"),level=warning)
```